src/renderer/components/ft-video-player/ft-video-player.vue

```diff
@@ -1,5 +1,30 @@
 <template>
   <div class="relative">
+    <transition
+      name="fade-blocked"
+    >
+      <div
+        v-if="videoBlocked && !unblockTemporarily"
+        class="blocked"
+        @click="handleUnblock"
+      >
+        <p class="blockedText">
+          {{ $t('Video.ChannelBlocker.Title') }}
+        </p>
+        <div class="blockedCountDown">
+          <p>{{ $t('Video.ChannelBlocker.Click to watch') }}</p>
+          <p
+            v-if="skipBlockedVideoCountDown >= 0"
+            :style="{visibility : skipBlockedVideoCountDown <= 60 ? 'visible' : 'hidden'}"
+          >
+            {{ $tc('Video.ChannelBlocker.Countdown Message', skipBlockedVideoCountDown, { countdown: skipBlockedVideoCountDown }) }}
+          </p>
+          <p v-else>
+            {{ $t('The playlist has ended.  Enable loop to continue playing') }}
```

Comment:
I don't believe this translation key exists

Reply:
It's under "Toast Messages"
https://github.com/FreeTubeApp/FreeTube/blob/48bdd3f284a7d52f35ffc34136f8af954c32d873/static/locales/en-US.yaml#L722

Reply:
Oups didn't notice that

Reply:
Should probably only use one space after the period though

Reply:
Yes and I'll unquote it too.